docs/source/correction.rst

```diff
@@ -0,0 +1,150 @@
+        will then by manifestly positive; however it is not guaranteed to be monotonic. If :math:`\nabla \Phi \sim M/r^2`, then
+
+        .. math::
+
+            \nabla_r M(<r) > 0 \implies 2r\nabla_r \Phi + r^2\nabla^2\Phi > 0.
+
+        This implies yet another (very rare) non-physicality:
+
+        .. admonition:: Type 2a Non-Physicalities
+
+            There is one Type 2a non-physicality which can arise in this case:
+
+            - **Type 2a-1a**: :math:`\nabla \Phi / \nabla \Phi_0 \le f(r)` (In Newtonian: :math:`f(r) = e^{-2r}`).
+
+        This Type 2a Non-Physicality is labeled as a Type 2 because it differs in its behavior between gravitational theories. With a monotonically increasing :math:`M(<r)`, it is guaranteed from
+        this formulation that :math:`M(<r)` will exceed or equal :math:`M_g(<r)`, thus concluding the construction of the model.
+
+    .. tab-item:: From :math:`\rho_g + \rho_{\mathrm{dyn}}`
+
+        The :math:`\rho_g + \rho_{\mathrm{dyn}}` approach is almost entirely self consistent. Only Type 0 non-physicalities occur
+        in this case.
+
+        .. admonition:: Trivial (Type 0) Non-Physicalities
+
+            Occur if any of the following are not true
+
+            - :math:`\rho_g < 0`
+            - :math:`\rho_{\mathrm{dyn}} < 0`
+            - :math:`\rho_g > \rho_{\mathrm{dyn}}`
+
+    .. tab-item:: From :math:`\rho_g + S_g`
+
+        The :math:`\rho_g + S_g` approach is entirely equivalent to the :math:`\rho_g+S_g` approach. Given that
+
+        .. math::
+
+            S(r) = k_bT_g(r)n_e(r)^{-2/3},
+
+        A temperature profile is immediately specified which must meet all of the criteria for the temperature profile in the :math:`\rho_g + T_g` approach.
+
+    .. tab-item:: Without gas
+
+        This approach is entirely self-consistent as long as the density profile provided is physical.
+
+Type 2 Non-Physicalities
+''''''''''''''''''''''''
+
+Type 2 Non-physicalities are gravity theory specific. Due to the modularity of the gravity implementation and the capacity for the CGP to support arbitrary
+gravitational theories, we will not exhaustively cover niche cases here. Documentation for each of the individual gravity theories should include a table with
+the relevant possible non-physicalities.
+
+As was described in the previous section, some fairly banal non-physicalities can be gravity theory dependent (Type 2a) but should not be interpreted as some "gravitational inconsistency." A second class of Type 2 non-physicalities exists,
+namely Type 2b, which includes non-physicality driven manifestly by the behavior of specific modified gravity theories. While these instances are also niche and should be found in the gravitational theory's documentation, one such instances is ubiquitous
+enough to deserve it's mention: Type 2b-1. These are called **Newtonian-MOND disequivalence errors** and occur due to the following theorem
```

Comment:
Is this a common term? Can you provide a reference for it?

Reply:
@jzuhone The underlying theorem (principle? Dan and I can't decide) is one of the new results in the paper Dan and I are working on finishing up. Entirely my own term, but I do think it's the most descriptive option.